subgraphs/_reference_/src/sdk/protocols/lending/manager.ts

```diff
@@ -227,6 +228,11 @@ export class DataManager {
     return this.market;
   }
 
+  updateLastMarketTimestamp(): void {
```

Comment:
alt: rename function to `saveMarket()` and replace everywhere there is a `this.market.save()`

Reply:
Let me fix that. It is only intended to be updated with supply/borrowIndex since that timestamp is needed to calculate delta between last snapshot and current index